datalake/src/main/java/com/sequenceiq/datalake/flow/create/handler/SdxValidationHandler.java

```diff
@@ -0,0 +1,87 @@
+package com.sequenceiq.datalake.flow.create.handler;
+
+import static com.sequenceiq.cloudbreak.common.exception.NotFoundException.notFound;
+
+import java.util.Optional;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import com.dyngr.exception.PollerException;
+import com.dyngr.exception.PollerStoppedException;
+import com.dyngr.exception.UserBreakException;
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.datalake.entity.SdxCluster;
+import com.sequenceiq.datalake.flow.create.event.SdxCreateFailedEvent;
+import com.sequenceiq.datalake.flow.create.event.SdxValidationRequest;
+import com.sequenceiq.datalake.flow.create.event.SdxValidationSuccessEvent;
+import com.sequenceiq.datalake.repository.SdxClusterRepository;
+import com.sequenceiq.datalake.service.sdx.EnvironmentService;
+import com.sequenceiq.datalake.service.sdx.SdxRecommendationService;
+import com.sequenceiq.environment.api.v1.environment.model.response.DetailedEnvironmentResponse;
+import com.sequenceiq.flow.reactor.api.handler.ExceptionCatcherEventHandler;
+import com.sequenceiq.flow.reactor.api.handler.HandlerEvent;
+
+import reactor.bus.Event;
+
+@Component
+public class SdxValidationHandler extends ExceptionCatcherEventHandler<SdxValidationRequest> {
```

Comment:
I know StorageValidationHandler is not covered with UT, but this requires UT

Reply:
Why? What is the difference?

Reply:
I don't see the missing unit test, could you point me to it or add it if it's still missing?

Reply:
I have created new unit tests for related @Service classes and I have created a new mock test as well, but I havent created test for the handler.

Reply:
could you create a test for the handler? thanks

Reply:
done